earth_enterprise/rpms/opengee-fusion/snippets/post-install.sh

```diff
@@ -150,10 +150,10 @@ install_or_upgrade_asset_root()
         if [ "$IS_SLAVE" = "false" ]; then
             OWNERSHIP=`find "$ASSET_ROOT" -maxdepth 0 -printf "%g:%u"`
             if [ "$OWNERSHIP" != "$GEGROUP:$GEFUSIONUSER" ] ; then
-                NOCHOWN=""
-                UPGRADE_MESSAGE="The upgrade will fix permissions for the asset root and source volume. This may take a while."
+                UPGRADE_MESSAGE="WARNING: The installer detected the asset root may have  incorrect permissions! \
+After installation you may need to run \n\n\
+$BASEINSTALLDIR_OPT/bin/geconfigureassetroot --noprompt --chown --assetroot $ASSET_ROOT\n\n"
```

Comment:
Turns out this command doesn't work. You have to include the `--repair` flag. The same likely applies to install_fusion.sh.

Strangely enough, the command for geconfigurepublishroot works fine.

Reply:
Minor: while you're fixing this, you might as well add `sudo` at the beginning of this command (and make the same change in install_fusion.sh).